<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationInfoDialog</class>
 <widget class="QDialog" name="StationInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_activeSensor">
       <property name="text">
        <string>active sensor name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_pointProperties">
       <property name="text">
        <string>show station point properties</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget_sensorConfig">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab_config">
      <attribute name="title">
       <string>configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
         <item>
          <widget class="QLabel" name="label_configName">
           <property name="text">
            <string>config name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_configName">
           <property name="placeholderText">
            <string>configuration name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_existingConfigs">
           <property name="text">
            <string>existing configs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_existingConfigs"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
         <item>
          <widget class="QToolButton" name="toolButton_loadConfig">
           <property name="text">
            <string>load configuration</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_saveConfig">
           <property name="text">
            <string>save configuration</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_accuracy">
      <attribute name="title">
       <string>accuracy</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolBox" name="toolBox_accuracy">
         <property name="currentIndex">
          <number>4</number>
         </property>
         <widget class="QWidget" name="page_polar">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>122</width>
            <height>96</height>
           </rect>
          </property>
          <attribute name="label">
           <string>polar</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_sigmaAzimuth">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_sigmaAzimuth"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_sigmaZenith">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_sigmaZenith"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_sigmaDistance">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_sigmaDistance"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_cartesian">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>122</width>
            <height>96</height>
           </rect>
          </property>
          <attribute name="label">
           <string>cartesian</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_sigmaX">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_sigmaX"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_sigmaY">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_sigmaY"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_sigmaZ">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_sigmaZ"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_temperature">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>122</width>
            <height>40</height>
           </rect>
          </property>
          <attribute name="label">
           <string>temperature</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_sigmaTemperature">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_sigmaTemperature"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_levelMeasurement">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>122</width>
            <height>68</height>
           </rect>
          </property>
          <attribute name="label">
           <string>level measurement</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_sigmaXZ">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_sigmaXZ"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_sigmaYZ">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_sigmaYZ"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_sigmaUndefined">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>558</width>
            <height>249</height>
           </rect>
          </property>
          <attribute name="label">
           <string>others</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_sensorConfig">
      <attribute name="title">
       <string>sensor configuration</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_connection">
      <attribute name="title">
       <string>connection</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_27" stretch="1,1">
         <item>
          <widget class="QLabel" name="label_connectionType">
           <property name="text">
            <string>connection type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_connectiontype"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_28" stretch="1,1">
         <item>
          <widget class="QLabel" name="label_ip">
           <property name="text">
            <string>ip</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QComboBox" name="comboBox_ip"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_ip">
             <property name="placeholderText">
              <string>put in ip adress if not found above</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_26" stretch="1,1">
         <item>
          <widget class="QLabel" name="label_port">
           <property name="text">
            <string>port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_port">
           <property name="placeholderText">
            <string>port</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_24" stretch="1,1">
         <item>
          <widget class="QLabel" name="label_comport">
           <property name="text">
            <string>comport</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_comport"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_29" stretch="1,1">
         <item>
          <widget class="QLabel" name="label_baudrate">
           <property name="text">
            <string>baudrate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_baudrate"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_30" stretch="1,1">
         <item>
          <widget class="QLabel" name="label_databits">
           <property name="text">
            <string>databits</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_databits"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_31" stretch="1,1">
         <item>
          <widget class="QLabel" name="label_flowcontrol">
           <property name="text">
            <string>flowcontrol</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_flowcontrol"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="1,1">
         <item>
          <widget class="QLabel" name="label_parity">
           <property name="text">
            <string>parity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_parity"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_25" stretch="1,1">
         <item>
          <widget class="QLabel" name="label_stopbits">
           <property name="text">
            <string>stopbits</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_stopbits"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_connectionChanged">
         <property name="text">
          <string>connection changed</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButton_cancel">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_ok">
       <property name="text">
        <string>apply changes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
